src/index.js

```diff
@@ -277,6 +298,7 @@ class CreditCardInput extends Component {
               pattern="[0-9]*"
               placeholder="CVC"
               type="text"
+              maxLength={3}
```

Comment:
American Express can be a 4 digit CVC, would be nice to dynamically set this accordingly. Can use [https://github.com/braintree/credit-card-type#code](https://github.com/braintree/credit-card-type#code) to get the code length of a card.

Reply:
or just make it 4 if it is amex lol

Reply:
NO way

Reply:
Didn't know that, nice catch!